cmd/commands/cli/command.go

```diff
@@ -356,6 +357,53 @@ func (c *cliApp) unlock(argsString string) {
 	success(fmt.Sprintf("Identity %s unlocked.", identity))
 }
 
+func (c *cliApp) payout(argsString string) {
+	const usage = "payout command:\n    set"
+	if len(argsString) == 0 {
+		info(usage)
+		return
+	}
+
+	args := strings.Fields(argsString)
+	if len(args) < 1 {
+		info(usage)
+		return
+	}
+
+	action := args[0]
+	switch action {
+	case "set": // Known sub-commands.
+	default:
+		warnf("Unknown sub-command '%s'\n", action)
+		fmt.Println(usage)
+		return
+	}
+
+	if action == "set" {
```

Comment:
This block can be a part of the `switch case "set"`, we don't need to check it twice.

Reply:
Moved to switch